### PR TITLE
DBZ-4794 Fix ddl parser test error

### PR DESCRIPTION
**Pull request by seekingua**
fix oracle ddl parser test error: [DBZ-4641](https://issues.redhat.com/browse/DBZ-4641), we can get 'create table' syntax for oracle 19 from: [Create Table](https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/CREATE-TABLE.html#GUID-F9CE0CC3-13AE-4744-A43C-EAC7A71AAAB6)

we should follow the order like this:
`inmemory_memcompress -> inmemory_priority -> inmemory_distribute -> inmemory_duplicate`

after change the order for inmemory table clause,  the tests for oracle has passed
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Debezium Build Aggregator 1.8.0-SNAPSHOT:
[INFO]
[INFO] Debezium Build Aggregator .......................... SUCCESS [  0.496 s]
[INFO] Debezium BOM ....................................... SUCCESS [  0.008 s]
[INFO] Debezium IDE Formatting Rules ...................... SUCCESS [  0.475 s]
[INFO] Debezium Checkstyle Rules .......................... SUCCESS [  0.799 s]
[INFO] Debezium Revapi Rules .............................. SUCCESS [  0.045 s]
[INFO] Debezium Parent POM ................................ SUCCESS [  0.247 s]
[INFO] Debezium API ....................................... SUCCESS [  2.318 s]
[INFO] Debezium Core ...................................... SUCCESS [ 34.543 s]
[INFO] Debezium ANTLR DDL parsers ......................... SUCCESS [ 35.253 s]
[INFO] Debezium Assembly Descriptors ...................... SUCCESS [  0.020 s]
[INFO] Debezium Embedded .................................. SUCCESS [ 14.421 s]
[INFO] Debezium Connector for MySQL ....................... SUCCESS [ 32.242 s]
[INFO] Debezium Schema Generator .......................... SUCCESS [  1.307 s]
[INFO] Debezium Connector for PostgreSQL .................. SUCCESS [  4.082 s]
[INFO] Debezium Connector for MongoDB ..................... SUCCESS [  3.355 s]
[INFO] Debezium Connector for MS SQL Server ............... SUCCESS [ 22.194 s]
[INFO] Debezium Scripting ................................. SUCCESS [  2.462 s]
[INFO] Debezium Microbenchmark Tests ...................... SUCCESS [  6.464 s]
[INFO] Debezium Quarkus :: Outbox ......................... SUCCESS [  0.014 s]
[INFO] Debezium Quarkus :: Outbox :: Runtime .............. SUCCESS [  1.581 s]
[INFO] Debezium Quarkus :: Outbox :: Deployment ........... SUCCESS [  0.636 s]
[INFO] Debezium Quarkus :: Outbox :: Integration Tests .... SUCCESS [ 36.763 s]
[INFO] Debezium Server BOM ................................ SUCCESS [  0.018 s]
[INFO] Debezium Testing ................................... SUCCESS [  0.010 s]
[INFO] Debezium Testing Testcontainers .................... SUCCESS [  1.346 s]
[INFO] Debezium Server Parent ............................. SUCCESS [  0.112 s]
[INFO] Debezium Server Core ............................... SUCCESS [ 20.548 s]
[INFO] Debezium Server Amazon Kinesis Sink Adapter ........ SUCCESS [  1.828 s]
[INFO] Debezium Server Google Cloud Pub/Sub Sink Adapter .. SUCCESS [  1.899 s]
[INFO] Debezium Server Pulsar Sink Adapter ................ SUCCESS [  1.894 s]
[INFO] Debezium Server Azure Event Hubs Adapter ........... SUCCESS [  1.567 s]
[INFO] Debezium Server Redis Sink Adapter ................. SUCCESS [  1.549 s]
[INFO] Debezium Server Distribution ....................... SUCCESS [  1.182 s]
[INFO] Debezium Server Kafka Sink Adapter ................. SUCCESS [  1.572 s]
[INFO] Debezium Server Pravega Sink Adapter ............... SUCCESS [  1.601 s]
[INFO] Debezium Server NATS Streaming Sink Adapter ........ SUCCESS [  1.531 s]
[INFO] Debezium system integration test-suite ............. SUCCESS [  2.037 s]
[INFO] Debezium Kafka Connect REST Extension .............. SUCCESS [  1.080 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  04:00 min
[INFO] Finished at: 2022-02-24T21:06:58+08:00
[INFO] ------------------------------------------------------------------------
```

### CODE REVIEW

**Comment by Naros** [comment]
Hi @seekingua so I actually believe this is a mistake in the backport of DBZ-4641 to the 1.8 branch. 

Would you please change the PR to remove that line from the SQL file, it shouldn't have been added.  Secondly, could you reword the commit message and use DBZ-4641 instead of DBZ-4794?

File: debezium-ddl-parser/src/test/resources/oracle/examples/ddl_create.sql [hunk 1]
```diff
@@ -18,7 +18,7 @@ CREATE TABLE T$BO_PATIENT_LMAP NOLOGGING TABLESPACE CMX_TEMP_TDE PARALLEL (DEGRE
...
 create table products (id NUMBER(4) NOT NULL PRIMARY KEY, name VARCHAR2(255) NOT NULL, seq NUMBER NOT NULL);
 CREATE TABLE TT_BSTOFF_VERDICHT ( TrxKey NUMBER(12), BStoffPKey NUMBER(12), BStoffBOId VARCHAR2(40), BStoffBelegNr NUMBER(12), BStoffBetrag NUMBER(15,2), BStoffBetragWA NUMBER(15,2),  PRIMARY KEY (TrxKey, BStoffPKey)) ORGANIZATION INDEX;
-CREATE TABLE "ODS_XMES_QY"."ORDER" ("Oid" CHAR(36), "CONTRACT" CHAR(36), CONSTRAINT "PK_ORDER" PRIMARY KEY ("Oid") USING INDEX ENABLE, SUPPLEMENTAL LOG GROUP "GGS_87678" ("Oid") ALWAYS) INMEMORY PRIORITY CRITICAL MEMCOMPRESS FOR QUERY LOW DISTRIBUTE AUTO FOR SERVICE ALL NO DUPLICATE;
+CREATE TABLE "ODS_XMES_QY"."ORDER" ("Oid" CHAR(36), "CONTRACT" CHAR(36), CONSTRAINT "PK_ORDER" PRIMARY KEY ("Oid") USING INDEX ENABLE, SUPPLEMENTAL LOG GROUP "GGS_87678" ("Oid") ALWAYS) INMEMORY MEMCOMPRESS FOR QUERY LOW PRIORITY CRITICAL DISTRIBUTE AUTO FOR SERVICE ALL NO DUPLICATE;
```
Comment (Naros) [inline debezium-ddl-parser/src/test/resources/oracle/examples/ddl_create.sql:21]:
```suggestion
```

**Review by Naros — CHANGES_REQUESTED** [review]

File: debezium-ddl-parser/src/test/resources/oracle/examples/ddl_create.sql (hunk 1 above)
Comment (seekingua) [reply debezium-ddl-parser/src/test/resources/oracle/examples/ddl_create.sql:21]: @Naros yeah ok, will remove that line

**Comment by Naros** [comment]
Thanks @seekingua, reworded the commit, rebased, and cherry-picked to 1.8.

**Pull request closed without merging**